### PR TITLE
arm: add install script, fix Makefile

### PR DESCRIPTION
**Pull request by zramsay**
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

**Comment by melekes** [comment]
```
#!/bin/bash -eo pipefail
set -ex
export PATH="$GOBIN:$PATH"
make metalinter
+ export PATH=/tmp/workspace/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ PATH=/tmp/workspace/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ make metalinter
--> Running linter
WARNING: exec: "deadcode": executable file not found in $PATH
WARNING: exec: "gosimple": executable file not found in $PATH
WARNING: exec: "misspell": executable file not found in $PATH
WARNING: exec: "safesql": executable file not found in $PATH
Makefile:231: recipe for target 'metalinter' failed
make: *** [metalinter] Error 2
Exited with code 2
```

File: Makefile [hunk 1]
```diff
@@ -79,8 +79,6 @@ check_tools:
 get_tools:
 	@echo "--> Installing tools"
 	./scripts/get_tools.sh
-	@echo "--> Downloading linters (this may take awhile)"
-	$(GOPATH)/src/github.com/alecthomas/gometalinter/scripts/install.sh -b $(GOBIN)
```
Comment (Liamsi) [inline Makefile]: If you add back these two lines, this should resolve https://github.com/tendermint/tendermint/pull/2824#issuecomment-438166072 AFAIK.

Comment (zramsay) [reply Makefile]: this line fails when installing on RPi: https://github.com/alecthomas/gometalinter/blob/master/scripts/install.sh#L62

should we add a platform check for this line?

Comment (melekes) [reply Makefile]: > should we add a platform check for this line?

yes (with a comment saying it does not work on ...)

Comment (zramsay) [reply Makefile]: the better option (seemingly) was to split the makefile targets - this is how it is in the SDK now

File: scripts/install/install_tendermint_arm.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# XXX: this script is intended to be run from
+# a fresh Digital Ocean droplet with Ubuntu
```
Comment (ebuchman) [inline scripts/install/install_tendermint_arm.sh]: Oh?

**Comment by zramsay** [comment]
@ebuchman yeah i was thinking the same thing ... it'll happen

**Comment by zramsay** [comment]
https://github.com/tendermint/coding/issues/72

**Pull request merged**